src/pk.c

```diff
@@ -267,8 +267,8 @@ static int wolfssl_print_number(WOLFSSL_BIO* bio, mp_int* num, const char* name,
     for (i = 0; (ret == 1) && (i < rawLen); i++) {
         /* Check for a complete line. */
         if (li >= PRINT_NUM_MAX_DIGIT_LINE) {
-            /* More bytes coming so append carriage return. */
-            line[li++] = '\n';
+            /* More bytes coming so change the trailing ':' to a line break. */
+            line[li-1] = '\n';
```

Comment:
Put a '\0' after as well rather than remove ':'.
':' indicates bytes on next line belong with this one.

Reply:
Thanks Sean -- I was concerned about that.  Will push fix shortly.

Reply:
done.  and much other goodness besides.  see 361156c703d9a69610f8099d6a9802b56fb4c746